widget/ui/src/components/SelectableCategoryList/index.ts

```diff
@@ -1,2 +1,3 @@
 export { SelectableCategoryList } from './SelectableCategoryList';
 export { BlockchainCategories } from './SelectableCategoryList.types';
```

Comment:
Based on the design the titles are not uppercase in some cases. Maybe it's their mistake:
![image](https://github.com/rango-exchange/rango-client/assets/123297941/522656d9-5ae4-44e7-8475-01d771357085)

Reply:
Done